docs/source/trude.rst

```diff
@@ -1,5 +1,109 @@
 Trude
 ==========
 
-.. caution::
-  This page is currently not created, we are working on it. If you would like to contribute on this documentation, reach `me <helena.almamol@gmail.com>`_!
+*From Germanic, Gertrud, hypocorism: female friend.*
+
+This city produces the light and dark spectrum of SiPMs for dedicated calibration runs. This is achieved by selecting regions in the SiPM waveforms where LED pulses are expected and regions which end 2 microseconds before, respectively and integrating the content within each region. The regions before LED pulses should only contain electronics noise and dark counts giving the zero external light approximation whereas those in time with the pulses will contain one or more detected photoelectrons. The waveform integrals are split into two groups: those with expected photoelectrons (light) and those without expected photoelectrons (dark). Each group produces a different spectrum.
+
+.. _Trude input:
+
+Input
+-----
+
+ * ``/Run/events``
+ * ``/Run/runInfo``
+ * ``/RD/sipmrwf``
+
+.. _Trude output:
+
+Output
+------
+
+* ``/HIST/sipm_dark``: histogram values of the dark spectrum.
+* ``/HIST/sipm_dark_bins``: bin  values of the dark spectrum.
+* ``/HIST/sipm_spe``: histogram values of the light spectrum.
+* ``/HIST/sipm_spe_bins``: bin  values of the light spectrum.
+
+.. _Trude config:
+
+Config
+------
+
+Besides the :ref:`Common arguments to every city`, *Trude* has the following arguments:
+
+.. list-table::
+   :widths: 50 40 120
+   :header-rows: 1
+
+   * - **Parameter**
+     - **Type**
+     - **Description**
+
+   * - ``min|max_bin``
+     - ``float``
+     - Lower/upper limit of the number of ACDs of the waveform to be considered for the spectrum.
+
+   * - ``bin_width``
+     - ``int``
+     - Width of the bins for the spectrum.
+
+   * - ``proc_mode``
+     - ``string``
+     - Two options of subtracting the baseline: ``subtract_mode`` to subtract the mode and ``subtract_median`` to subtract the median.
+
+   * - ``number_integrals``
+     - ``int``
+     - Number of integrals to be performed in the waveform's time window. Typically this value is selected depending on the number of LED pulses in the waveform.
+
+   * - ``integral_start``
+     - ``int``
+     - Bin where the integrals start.
```

Comment:
Maybe I would specify here that it is the bin where the first integral starts, so that then the other ones are integrated using the `integrals_period` by assuming the same separation between them.

Reply:
Done!